src/shared/ui/Skeleton/README.md

```
@@ -0,0 +1,156 @@
+
+// 원형 (프로필 이미지 등)
+<Skeleton.Circle width={40} />
+
+// 텍스트 라인
+<Skeleton.Text lines={3} height={16} gap={8} />
+```
+
+## API
+
+### Skeleton (기본)
+| Props | Type | Default | Description |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)